deploy/scripts/deploy_quadlets.sh

```
@@ -59,6 +59,62 @@ if sudo podman exec flightctl-db psql -U postgres -tAc "SELECT rolsuper FROM pg_
 
+# Set up database users for production deployment
+echo "Setting up database users for production deployment..."
+DB_MIGRATION_PASSWORD=$(sudo podman exec flightctl-db printenv POSTGRESQL_MIGRATOR_PASSWORD 2>/dev/null || echo "migrator_password")
+DB_APP_PASSWORD=$(sudo podman exec flightctl-db printenv POSTGRESQL_PASSWORD 2>/dev/null || echo "app_password")
+
+# Set environment variables for the database setup script
+export DB_HOST=flightctl-db
+export DB_PORT=5432
+export DB_NAME=flightctl
+export DB_ADMIN_USER=admin
+export DB_ADMIN_PASSWORD=$(sudo podman exec flightctl-db printenv POSTGRESQL_MASTER_PASSWORD)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)